float32 infrared_left
float32 infrared_right
float32 ultrasound_left
float32 ultrasound_right


